src/popups/HandicapPopup/HandicapPopup.scss

```diff
@@ -1,10 +1,37 @@
 .wkp-handicap-popup {
   .wkp-handicap-popup-title {
+    background-color: #f5f5f5;
     font-weight: bold;
-    margin: 10px 0 5px 0;
+    position: absolute;
```

Comment:
why absolute positioning?

Reply:
To position this header on the top of the popup.

Reply:
Also related to the css of react-spatial-popup. Will be fixed when we clean react-spatial.